src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -622,142 +622,324 @@ static GetItemEntry sGetItemTable[] = {
     GET_ITEM_NONE,
 };
 
-static LinkAnimationHeader* D_80853914[PLAYER_ANIMGROUP_MAX][PLAYER_ANIMTYPE_MAX] = {
+#define PLAYER_ANIM_BY_GROUP_AND_TYPE(group, type) D_80853914[group * PLAYER_ANIMTYPE_MAX + type]
```

Comment:
This is a bit long for what it does imo, I think it could be shortened to something simpler like `GET_PLAYER_ANIM`. The macro itself contains names for the arguments already, and it would usually be called with enum values and/or variables named "group" or "animType", so I think it would be fine.

Also I believe this needs a formatter run (at least with the current name)

Reply:
Surprisingly the PR is already correctly formatted (I already ran the formatter), so the name isn't that long I guess. it helped me going with this name even if it's quite long

I avoided giving it a generic name such as `get_player_anim` as it felt too generic and this isn't the only thing for handling link's animations, but maybe it's okay?

Reply:
I feel like it would be fine to name it with a more generic name now and rename it later if it ever starts conflicting or risks being confused with other macros. A better name will probably be easier to figure out at that point as well, since we'll know more about the other ways to define link's animations and how they differ from this one